torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -163,19 +145,34 @@ def pad_addmm(
                 input = pad_dim(input, n_padded_length, 1)
             elif input.dim() == 1 and input.shape[0] != 1:
                 input = pad_dim(input, n_padded_length, 0)
-        elif m_padded_length != 0 and input.dim() == 2 and input.shape[0] != 1:
+        if m_padded_length != 0 and input.dim() == 2 and input.shape[0] != 1:
```

Comment:
Now that I remember: i believe the regression was because we were expanding the one D bias in a way that we would no longer hit cublas addmm. in any case - multi dimensional bias is extremely uncommon. and ive added tests here for all of the difference ways bias might be hit